testing-guidance/workflow-tests/scripting/script-canvas-tests.md

```diff
@@ -5,53 +5,57 @@ Testing in this area should focus on the functionality of Script Canvas editor.
+| Script Canvas editor tools can be opened and configured  | <ol><li>Open each of the Script Canvas editor tools.</li><li>Click and try to drag every UI item of the opened tool.</li></ol>                                                     | <ul><li>Script Canvas tools render properly.</li><li>Tools do not become unresponsive after configuring them.</li><li>UI elements in each of the tools respond when clicked or dragged</li>                                                                                                                                                                   |
+| Using basic Script Canvas graph controls | <ol><li>Create a new script canvas graph file, add a few nodes and perform every action from:</li><ul><li>View menu</li><li>Edit Menu</li><li>File menu</li></ul></ul></li><li>Perform every control action located in Script Canvas graph window top left corner.</li></ol> | <ul><li>Every control action and action from Script Canvas menu bar perform corresponding actions.</li><li>Every control action and action from Script Canvas menu can be performed with associated keyboard shortcut.</li><li>Controls not responding or responding too slowly when used.</li><li>Controls gray out under appropriate circumstances.</li><li>Grayed out controls can not be interacted with</li><li>A helpful tooltip is generated and rendered properly for each control.</li><li>Tooltips do not go away prematurely.</li><li>Script Canvas graph renders properly after Pan/Zoom actions.</li></ul>                                                     
+| Opening a Script Canvas graph from older version of the engine            | <ol><li>Obtain a script canvas graph file from an earlier version of O3DE , put it into used project directory and open it with Script Canvas editor.</li><li>Following graph file [OldGraphTest.scriptcanvas](https://github.com/o3de/sig-content/raw/main/testing-guidance/workflow-tests/scripting/scripts/OldGraphTest.scriptcanvas) derived from Development (446de8a) build on 1/10 can be used for this test.</li></ol> | <ul><li>Graph from the older version of the engine is processed by Asset Processor without errors and can be opened with Script Canvas editor.</li><li>If [OldGraphTest.scriptcanvas](https://github.com/o3de/sig-content/raw/main/testing-guidance/workflow-tests/scripting/scripts/OldGraphTest.scriptcanvas) was used and assigned to entity upon entering the game mode/simulation mode a message reading _Old Graph test successful_ should appear in the console.</li> |
```

Comment:
@LB-MateuszSzweda looks like you just did a raw link to the SHA by doing a `(text)[github.com/SHA-LINK]` vs using the MD trick of doing `o3de/o3de@SHA` (note, this works for `o3de/sig-name@SHA` as well.

For this SHA you would do `o3de/o3de@446de8a`.  

Let's use the Github convention here since these docs live in the o3de/sig-content repository and could lead somebody to look for changes in the wrong location.

Reply:
@FuzzyCarterAWS  I've done it because sadly no matter how I've formatted 
o3de/o3de@446de8a
o3de/o3de@446de8ac0fb3d5e803a7bf32f33fd04f50328de6

Inputting it in md. file always resulted in a non linked text in a preview. It looks like that function is supported in comments but not the md. files.

I've tried enclosing it in < > but even though it looked correct in a preview it was treated as an e-mail when clicked.

I had to revert that < > change and decided to delete the direct build number listing altoghether because 
(text)[github.com/SHA-LINK] was the only one that allowed linking.

Reply:
@LB-MateuszSzweda It looks like you're correct. For some reason I was under the impression that it worked when rendered on GitHub.

Reply:
In the future, if you wish to link to a SHA link, we can use the following pattern `[o3de/o3de@SHA](github.com/SHA-LINK)